src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -199,17 +197,15 @@ export const SPECIAL_CURRENCY_INFO: {
     displayBuyCrypto: true,
     displayIoniaRewards: true,
     isImportKeySupported: true,
-    isStakingSupported: true,
-    isPaymentProtocolSupported: true
+    isStakingSupported: true
   },
   bitcointestnet: {
     hasSegwit: true,
     maxSpendTargets: UTXO_MAX_SPEND_TARGETS,
     initWalletName: lstrings.string_first_bitcoin_testnet_wallet_name,
     chainCode: 'TESTBTC',
     displayBuyCrypto: true,
-    isImportKeySupported: true,
-    isPaymentProtocolSupported: true
```

Comment:
There are a bunch of currency codes that are not in the new `PAYMENT_PROTOCOL_MAP`:

- bitcointestnet
- bitcoinsv
- digibyte
- dash
- ...

I see that BitPay does not support these, and I don't know of other service providers that use the payment protocol, so there isn't much we can do about it. We are "losing support" for these formerly-supported coins, but I guess it's fine because they were never supported in the first place.